lib/expired_record_cleaner.rb

```diff
@@ -37,24 +37,30 @@ def notify_proposal_requester(proposal)
   end
 
   def handle_no_requester(proposal)
-    if @verbose
+    if @verbose && ENV["RAILS_ENV"] != "test"
```

Comment:
I am not sure why. I guess I was thinking we might want to run the specs with the env var changed, but `Rails.env` wraps the env var so it is basically the same thing. Will change.